bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,289 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affects_comments: bool=True,
+                        space_before_and_after_arrow: bool=False,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=0,
+                        spacing_in_empty_braces: int=0,
+                        camelcase_classes_name: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        check_whether_file_ends_in_newline: bool=False,
```

Comment:
okay :)

2016-07-09 9:18 GMT+01:00 Lasse Schuirmann <notifications@github.com>:

> In bears/coffee_script/CoffeeLintBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/555#discussion_r70164262>
> :
>
> > -        return '--reporter=raw', filename
> > +        return '--reporter=raw', '-f', config_file, filename
> > +
> > +    @staticmethod
> > +    def generate_config(filename, file,
> > +                        max_line_length: int=79,
> > +                        max_line_length_affects_comments: bool=True,
> > +                        space_before_and_after_arrow: bool=False,
> > +                        check_braces_spacing: bool=False,
> > +                        braces_spacing_width: int=0,
> > +                        spacing_in_empty_braces: int=0,
> > +                        camelcase_classes_name: bool=True,
> > +                        spaces_before_and_after_colon: bool=False,
> > +                        spaces_before_colon: int=0,
> > +                        spaces_after_colon: int=0,
> > +                        check_whether_file_ends_in_newline: bool=False,
>
> there's a setting for this in the SpaceConsistencyBear, can you use the
> same name here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/555/files/fe89d424248e4e93fc742cc09665aa173771d3fa#r70164262>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xojxGULAMmgnPwBWaIk7tNvbGYl0xks5qT1lEgaJpZM4I9f6t>
> .
>